cloud-aws/src/main/resources/definitions/datahub-minimal-policy.json

```diff
@@ -0,0 +1,131 @@
+        "ec2:StartInstances",
+        "ec2:StopInstances",
+        "ec2:AttachVolume",
+        "ec2:DescribeAddresses",
+        "ec2:TerminateInstances",
+        "ec2:DeleteLaunchTemplate",
+        "ec2:DeleteSecurityGroup",
+        "autoscaling:DeleteAutoScalingGroup"
+      ],
+      "Resource": "*",
+      "Condition": {
+        "StringLike": {
+          "ec2:ResourceTag/Cloudera-Resource-Name": [
+            "crn:cdp:*"
+          ]
+        }
+      }
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:CreateTags",
+        "cloudformation:CreateStack"
+      ],
+      "Resource": [
+        "*"
+      ],
+      "Condition": {
+        "StringLike": {
+          "aws:RequestTag/Cloudera-Resource-Name": [
+            "crn:cdp:*"
+          ]
+        }
+      }
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:ReleaseAddress",
+        "cloudformation:DeleteStack",
+        "autoscaling:SuspendProcesses",
+        "autoscaling:UpdateAutoScalingGroup",
+        "autoscaling:ResumeProcesses",
+        "autoscaling:DetachInstances"
+      ],
+      "Resource": [
+        "*"
+      ],
+      "Condition": {
+        "StringLike": {
+          "aws:ResourceTag/Cloudera-Resource-Name": [
+            "crn:cdp:*"
+          ]
+        }
+      }
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeVolumes",
+        "ec2:DescribeInstances",
+        "ec2:RunInstances",
+        "ec2:DescribeRouteTables",
+        "ec2:DescribeImages",
+        "ec2:CreateSecurityGroup",
+        "ec2:CreateVolume",
+        "ec2:AuthorizeSecurityGroupIngress",
+        "ec2:DescribeAvailabilityZones",
+        "ec2:DescribeRegions",
+        "ec2:DescribeInstanceTypeOfferings",
+        "ec2:DescribeInstanceTypes",
+        "ec2:DescribeSubnets",
+        "ec2:DescribeKeyPairs",
+        "ec2:DescribeVpcs",
+        "ec2:DescribeSecurityGroups",
+        "ec2:DescribeLaunchTemplates",
+        "ec2:CreateLaunchTemplate",
+        "ec2:CreatePlacementGroup",
+        "ec2:DeletePlacementGroup",
+        "ec2:DescribePlacementGroups",
+        "autoscaling:CreateAutoScalingGroup",
+        "autoscaling:DescribeScalingActivities",
+        "autoscaling:DescribeAutoScalingGroups",
+        "cloudwatch:DescribeAlarms",
+        "cloudformation:DescribeStackEvents",
+        "cloudformation:DescribeStackResources",
+        "cloudformation:DescribeStackResource",
+        "cloudformation:DescribeStacks",
+        "elasticloadbalancing:DeregisterTargets",
+        "kms:ListKeys",
+        "kms:ListAliases",
+        "sts:DecodeAuthorizationMessage",
+        "s3:GetBucketLocation"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "iam:PassRole"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "iam:ListInstanceProfiles",
+        "iam:SimulatePrincipalPolicy"
```

Comment:
I wonder if this is really needed for Data Hub. Is this policy supposed to be a superset of the Data Lake policy? If not, `iam:SimulatePrincipalPolicy` is probably only used at environment and Data Lake level, but this shall be double checked in the code. 😃